### PR TITLE
Admin, Banner Manager: replace <?php echo

### PR DESCRIPTION
**Pull request by torvista**
...prior to some other items.

FYI
done with
search /.\<\?php echo (.*); \?>/m	
replace <?= $1 ?>

### CODE REVIEW

File: admin/banner_manager.php [hunk 1]
```diff
@@ -351,127 +351,127 @@ function popupImageWindow(url) {
...
             </div>
           </div>
           <div class="form-group">
-            <?php echo zen_draw_label(TEXT_BANNERS_GROUP, 'banners_group', 'class="col-sm-3 control-label"'); ?>
+            <?= zen_draw_label(TEXT_BANNERS_GROUP, 'banners_group', 'class="col-sm-3 control-label"') ?>
             <div class="col-sm-9 col-md-6">
-              <?php echo zen_draw_pull_down_menu('banners_group', $groups_array, $bInfo->banners_group, 'class="form-control" id="banners_group"'); ?><br>
-              <p><?php echo TEXT_BANNERS_NEW_GROUP; ?></p><?php echo zen_draw_input_field('new_banners_group', '', 'class="form-control" id="new_banners_group"', count($groups_array) === 0); ?>
+              <?= zen_draw_pull_down_menu('banners_group', $groups_array, $bInfo->banners_group, 'class="form-control" id="banners_group"') ?><br>
+              <p><?= TEXT_BANNERS_NEW_GROUP; ?></p><?php echo zen_draw_input_field('new_banners_group', '', 'class="form-control" id="new_banners_group"', count($groups_array) === 0) ?>
```
Comment (drbyte) [inline admin/banner_manager.php:398]: Looks like the regex pattern missed cases where multiple exist on the same line.

File: admin/banner_manager.php [hunk 2]
```diff
@@ -351,127 +351,127 @@ function popupImageWindow(url) {
...
+              <span class="help-block errorText">(<?= zen_datepicker_format_full(); ?>) <span class="date-check-error"><?php echo ERROR_INVALID_SCHEDULED_DATE ?></span></span>
             </div>
           </div>
           <div class="form-group">
-            <?php echo zen_draw_label(TEXT_BANNERS_EXPIRES_ON, 'expires_date', 'class="col-sm-3 control-label"'); ?>
+            <?= zen_draw_label(TEXT_BANNERS_EXPIRES_ON, 'expires_date', 'class="col-sm-3 control-label"') ?>
             <div class="col-sm-9 col-md-6">
               <div class="date input-group" id="datepicker_expires_date">
                 <span class="input-group-addon datepicker_icon">
                   <?php echo zen_icon('calendar-days', size: 'lg') ?>
```
Comment (drbyte) [inline admin/banner_manager.php:449]: Appears to also not replace when there's already no closing semicolon

**Comment by drbyte** [comment]
lol

But my point was that there are dozens more spots where those two formats are not updated in this PR.

Github syntax-highlights the "echo" keyword, and you can see those in the Files Changed view.

**Comment by torvista** [comment]
that was remarkably sloppy work, guess it's the summer heat, finally.

**Pull request closed without merging**